src/style/navbar.module.css

```diff
@@ -0,0 +1,37 @@
+.page-header {
```

Comment:
nit: but we usually call the highest level class just `container`. I'm guessing this is just what we called it in cfe? Alternatively `nav-bar` would be more consistent with the component name